src/main/java/no/unit/nva/model/exceptions/MalformedContributorException.java

```diff
@@ -0,0 +1,7 @@
+package no.unit.nva.model.exceptions;
+
+public class MalformedContributorException extends Exception {
```

Comment:
Do you want this very specific exception to be handled locally during stream processing? If yes then Exception is ok, but if you want it to bubble up to main exception handler then inherit from RuntimeException.

Reply:
We need to have a very concrete relationship with the exceptions here, I guess throwing them and throwing them again leaves a clearer trail than an unchecked exception. We need to discuss this further.

Reply:
Exceptions that we know how to handle should be checked, the rest should generally be unchecked.

Reply:
Sending a HTTP code is a way of handling, so if we anticipate that something can wrong, the it should be a checked Exception. Not handling is equivalent with returning Internal Server Error